Doc-11534-PM-2.8-Edits-DH  Partner Manager 2.8 edits (#141)
mulesoft/docs-partner-manager: modules/ROOT/Edits/create-partner.adoc

= Create a New Partner

> Create a new partner by using the *Partners* page to specify the partner's name, identifiers, endpoints, validation rules, and other settings.

Reviewer: I'm not sure that "other settings" is parallel with the specific items listed before it:
```suggestion
Create a new partner by using the *Partners* page to specify the partner's name, identifiers, endpoints, validation rules, and other specifics.
```
OR
```suggestion
Create a new partner by using the *Partners* page to specify partner-specific information such as name, identifiers, endpoints, and validation rules.
```

== Prerequisites

Before you create your first partner, xref:setup-payload-storage-API.adoc[make sure that you have configured your payload storage API].

> Obtain the following information:

Reviewer: Line 9 reads like an extension of Line 7, so how about...:
```suggestion
Before you create your first partner, xref:setup-payload-storage-API.adoc[make sure that you have configured your payload storage API] and that you have the following information:

Obtain the following information:
```

* The partner's name (usually the name of the company)
* One or more partner identifiers (such as the AS2 ID, X12-ISA, X12 GS, and so on)

== Create the Partner

After meeting the prerequisites to create a partner:

. In Partner Manager, select *Partners* in the navigation menu.
. Click *Create partner > Partner*.
. Enter the following information:
+
[%header%autowidth.spread]
|===
| Setting a| Description
| *Name*

> | Text string that easily identifies your partner

Reviewer: Do we need "easily"? Maybe we need "short" or "intuitive"?

| *Type*
| Identifier type
| *Qualifier*
| Value that appears if you selected *X12-ISA* or *EDIFACT-UNB* as the type
| *Identifier Value*

> | Value that uniquely identifies your partner

Reviewer: How is this different from the name?

|===
+
. Click *Create partner*.

== Configure Additional Partner Settings

> Add the following settings for the new partner, either now or later on:

Reviewer: ```suggestion
After you create the partner, add the following settings at any time prior to deployment:
```

Author: PM does not want to use the edited text.

[%header%autowidth.spread]
|===
|Settings |Action a|Additional Information
|Information about the partner, such as the partner's address, website, and contact details.
| Click *Edit* in the *About* section.
|
| Additional identifiers to associate with the partner.
| Click *New* in the *Identifiers* section.
| xref:partner-manager-identifiers.adoc[Sender and Receiver Identifiers]
| The partner's digital certificates, such as X.509 certificates.
| Click *New* in the *Certificates* section.
|

> | The partner-specific endpoints to use for sending and receiving messages to and from your partner, including acknowledgement endpoints. If you plan to use host endpoints only, you don't need to create partner-specific endpoints.

Reviewer: I wonder if we can reword this one item to be a fragment, so that we need not end all the rest in periods....

| Click *New* in the *Endpoints* section.
| xref:create-endpoint.adoc[Create and Configure Endpoints]
| Acknowledgements, validations, and control numbers for each message format that the partner uses. For X12, you can also configure character sets and separators (X12 send format only).
| Click the links in the *Validation and acknowledgement* settings.
a|
* xref:edifact-receive-read-settings.adoc[EDIFACT Receive Settings]
* xref:edifact-send-settings.adoc[EDIFACT Send Settings]
* xref:x12-receive-read-settings.adoc[X12 Receive Settings]
* xref:x12-send-settings.adoc[X12 Send Settings]
|===

== Next Step

> Once you have created a partner and configured partner-level endpoints (if needed), you can set up message flows.

Reviewer: GLOBAL, per style ref: once > after

Reviewer: Also...present tense
```suggestion
After you create a partner and configure partner-level endpoints (if needed), set up message flows.
```
